Lib/fontbakery/profiles/iso15008.py

```diff
@@ -147,12 +155,20 @@ def com_google_fonts_check_iso15008_stem_width(ttFont):
 @check(
     id = 'com.google.fonts/check/iso15008_intercharacter_spacing',
     rationale = """
-        According to ISO 15008, fonts used for in-car displays should not be too narrow or too wide.
-        To ensure legibility of this font on in-car information systems it is recommended that the spacing falls within the following values:
-        * space between vertical strokes (e.g. "ll") should be 150%-240% of the stem width.
-        * space between diagonals and verticals (e.g. "vl") should be at least 85% of the stem width.
-        * diagonal characters should not touch (e.g. "vv")."""
-    + DISCLAIMER,
+        According to ISO 15008, fonts used for in-car displays should not
+        be too narrow or too wide.
+        
+        To ensure legibility of this font on in-car information systems,
+        it is recommended that the spacing falls within the following values:
+        
+        * space between vertical strokes (e.g. "ll") should be 150%-240%
+          of the stem width.
+        
+        * space between diagonals and verticals (e.g. "vl") should be
+          at least 85% of the stem width.
+        
+        * diagonal characters should not touch (e.g. "vv").
+    """ + DISCLAIMER,
     proposal = ['https://github.com/googlefonts/fontbakery/issues/1832',
```

Comment:
At #3717 @miguelsousa said:
> I can look into it if you have an example. 

Here's an example of adding blank lines to make list itens to not end up all in the same line on the text terminal output. I am not sure if that's related to markdown parsing (and @aurium's usage of the `rich` library) or if that's a bug somewhere else in our handling of rationale text strings.

Reply:
To be honest, right I don't remember if `rich` was also used on the rationale text strings or if that was only used for nicer rendering of the check log messages. I gotta take some time to re-read that portion of the source to figure it out.

Reply:
Ah, that's what you mean.
This is exactly why I [added support soft-return in rationale strings](https://github.com/googlefonts/fontbakery/pull/3717) 😁 

When you mentioned bullet lists not rendering correctly, I thought you were referring to them not displaying like in this sample from rich's documentation:

<img width="994" alt="rich_sample" src="https://user-images.githubusercontent.com/2119742/163878329-bd4ba68f-f937-4ba0-96c7-6eac54e0fec1.png">

Reply:
The fix for these cases is simple:
1. Remove the extra blank line
2. Add a soft-return to the end of the previous line